common/web/lm-worker/src/main/predict-helpers.ts

```diff
@@ -278,7 +311,7 @@ export async function correctAndEnumerate(
 
     // Only set 'best correction' cost when a correction ACTUALLY YIELDS predictions.
     if(predictions.length > 0 && bestCorrectionCost === undefined) {
-      bestCorrectionCost = -Math.log(predictionRoot.p);
+      bestCorrectionCost = rootCost;
```

Comment:
Refer to line 307; it's the same value, just without the `exp` / `log` round-trip.